test/lib/jsonStathamSpec.js

```diff
@@ -2,120 +2,112 @@ import utils from '../../test/support/TestUtils'
+    it('builds get request', () => {
       jsonStatham.get('/bla')
       expect(args().url).to.equal('http://test.com/api/bla')
     })
 
-    it('builds PUT request', function() {
+    it('builds PUT request', () => {
       jsonStatham.put('/bla', {param: 'some data'})
       expect(args().url).to.equal('http://test.com/api/bla')
       expect(args().method).to.equal('PUT')
       expect(args().data.param).to.equal('some data')
     })
 
-    it('builds POST request', function() {
+    it('builds POST request', () => {
       jsonStatham.post('/bla', {param: 'some data'})
       expect(args().url).to.equal('http://test.com/api/bla')
       expect(args().method).to.equal('POST')
       expect(args().data.param).to.equal('some data')
     })
 
-    it('builds delete request', function() {
+    it('builds delete request', () => {
       jsonStatham.delete('/bla', {param: 'some data'})
       expect(args().url).to.equal('http://test.com/api/bla')
       expect(args().method).to.equal('DELETE')
       expect(args().data.param).to.equal('some data')
     })
 
-    it('sets auth credentials', function() {
+    it('sets auth credentials', () => {
       expect(jsonStatham.buildRequest('/bla', null, {}, true).xhrFields.withCredentials).to.equal(true)
     })
 
-    it('does not set auth credentials', function() {
+    it('does not set auth credentials', () => {
       expect(jsonStatham.buildRequest('/bla').xhrFields).to.equal(undefined)
     })
 
-    it('sends post with auth credentials', function() {
+    it('sends post with auth credentials', () => {
       jsonStatham.post('/bla', {email: 'dude@dude.com'}, true)
       expect(args().xhrFields.withCredentials).to.equal(true)
     })
   })
 
-  describe('requests: ', function () {
-    beforeEach(function() {
-      server = sinon.fakeServer.create()
-    })
-
-     afterEach(function() {
-      server.restore()
-    })
-
-    it('GET json', function(done) {
+  describe('requests: ', () => {
+    it('GET json', done => {
       utils.createServerAndMock('GET', '/api/test', JSON.stringify({fun: 'times'}), server, 200)
 
-      jsonStatham.fetch('/test').done(function(data) {
+      jsonStatham.get('/test').done(data => {
         expect(data.fun).to.equal('times')
         done()
       })
     })
 
-    it('POST data', function(done) {
+    it('POST data', done => {
       utils.createServerAndMock('POST', '/api/test', JSON.stringify({fun: 'times for post'}), server, 200)
 
-      jsonStatham.post('/test').done(function(data) {
+      jsonStatham.post('/test').done(data => {
         expect(data.fun).to.equal('times for post')
         done()
       })
     })
 
-    it('POST data with error', function(done) {
+    it('POST data with error', done => {
       utils.createServerAndMock('POST', '/api/test', JSON.stringify({error: {message: 'oops'}}), server, 422)
-      jsonStatham.post('/test').done(function() {}).fail(function(data) {
+      jsonStatham.post('/test').done(() => null).fail(data => {
```

Comment:
ESLint complains about both the use of `function` and empty arrow function bodies, so I return `null` here to shut it up. I confirmed that the tests still pass.